Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -478,6 +478,8 @@ class GraphicsLayer : public RefCounted<GraphicsLayer> {
 
     virtual void setContentsNeedsDisplay() { };
 
+    virtual void markDamageRectsUnreliable() { };
```

Comment:
I wouldn't call it “unreliable” 😉 

Probable something like `markCoveredByTransformedContent()` is more descriptive of what the flag tracks.

Reply:
The thing is, current implementation of that "unreliable" flag works like latch, so once the flag is set for the layer, it's always `true`. Renaming that to something like `markCoveredByTransformedContent` would require us to implement state changes which is not required from damaging perspective. Moreover, this function is rather temporary - should be removed as soon as we sort out the problems with 3d transforms and scrolling.